pandas/core/common.py

```diff
@@ -120,7 +120,7 @@ def is_bool_indexer(key):
     return False
 
 
-def _default_index(n):
+def default_index(n):
```

Comment:
this entire module, is technically public, though we have a mix of public/private here anyway, so have to be careful about changing public functions here (obviously this is private -> public).

Reply:
let' move this to ``pandas.index.base`` , ok to call it ``default_index``

Reply:
done